src/lib/onboard/managed-bootstrap/docker-test-fixture.ts

```
@@ -0,0 +1,483 @@
+            case args.includes("--shared-state-transaction-status"):
+              return ok(`${sharedState}\n`);
+            case args.includes("--rollback-shared-state-transaction"):
+              sharedState = "none";
+              events.push("shared:rollback");
+              return ok();
+          }
+          break;
+        case "exec":
+          switch (true) {
+            case args.includes("--commit-shared-state-transaction"):
+              const result = options.sharedStateCommitResult ?? ok();
```

Biome (2.5.5)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

(lint/correctness/noSwitchDeclarations)